lib/ansible/modules/cloud/amazon/lightsail_lb.py

```diff
@@ -0,0 +1,272 @@
+    instancePort = module.params.get('port')
+    healthCheckPath = module.params.get('healthCheckPath')
+    attachInstances = module.params.get('attachInstances')
+
+    resp = None
+    if loadbalancer is None:
+        try:
+            resp = client.create_load_balancer(
+                loadBalancerName=loadBalancerName,
+                instancePort=instancePort,
+                healthCheckPath=healthCheckPath
+            )
+            resp = resp['operations'][0]
+        except botocore.exceptions.ClientError as e:
+            module.fail_json(msg='Unable to create the load balancer {0}, error: {1}'.format(loadBalancerName, e))
+        changed = True
+
+        if attachInstances is not None:
+            try:
+                for instanceName in attachInstances:
+                    find_instance_info(client, instanceName)
+
+                attachedResp = client.attach_instances_to_load_balancer(
+                    loadBalancerName=loadBalancerName,
+                    instanceNames=attachInstances
+                )
+                attachedResp = attachedResp['operations'][0]
+            except botocore.exceptions.ClientError as e:
+                module.fail_json(msg='Unable to attach {0}, error: {1}'.format(loadBalancerName, e))
+
+    loadbalancer = get_load_balancer_info(client, loadBalancerName)
+    return (changed, loadbalancer)
+
+
+def delete_load_balancer(module, client, lb_name):
+    """
+    Terminates the load balancer
+
+    module: Ansible module object
+    client: authenticated lightsail connection object
+    lb_name: name of load balancer to delete
+
+    Returns a dictionary of the load balancer information
+    """
+    changed = False
+
+    loadbalancer = None
+    try:
+        loadbalancer = get_load_balancer_info(client, lb_name)
+    except botocore.exceptions.ClientError as e:
+        if e.response['Error']['Code'] != 'NotFoundException':
+            module.fail_json(msg='Error finding load balancer {0}, error: {1}'.format(lb_name, e))
+
+    if loadbalancer is not None:
+        try:
+            client.delete_load_balancer(loadBalancerName=lb_name)
+            changed = True
+        except botocore.exceptions.ClientError as e:
+            module.fail_json(msg='Error deleting load balancer {0}, error: {1}'.format(lb_name, e))
+
+    return (changed, loadbalancer)
+
+
+def core(module):
+    region, ec2_url, aws_connect_kwargs = get_aws_connection_info(module, boto3=True)
+    if not region:
+        module.fail_json(msg='region must be specified')
+
+    client = None
+    try:
+        client = boto3_conn(module, conn_type='client', resource='lightsail',
+                            region=region, endpoint=ec2_url, **aws_connect_kwargs)
+    except (botocore.exceptions.ClientError, botocore.exceptions.ValidationError) as e:
+        module.fail_json(msg='Failed while connecting to the lightsail service: %s' % e, exception=traceback.format_exc())
```

Comment:
With AnsibleAWSModule everything up to here can be replaced by `client = module.client('lightsail')`

Reply:
Done. #49974